### PR TITLE
Minor UI Tweaks 

### PR DESCRIPTION
**Pull request by litopia**
- Hooked up Select Genes/variables for a New Workbook on site navigation
- Cleaned up variable edit template
- Added initial state in Genes list template
- Changed dashboard color theme

### CODE REVIEW

File: templates/share/site_header.html [hunk 1]
```diff
@@ -260,7 +260,7 @@
                     {% endif %}
                     <li class="menu-item"><a href="{% url 'variables' %}">Manage Variables Favorite ( {{ user.variablefavorite_last_view_set.all|active:'variablefavorite'|length }} )</a></li>
                     <li class="menu-item"><a href="{% url 'variable_fav_create' %}">Create Favorite Variable(s)</a></li>
-                    <li class="menu-item"><a href="/">Select Variables for a New Workbook</a></li>
+                    <li class="menu-item"><a href="{% url 'select_and_create_workbook' %}">Select Variabless for a New Workbook</a></li>
```
Comment (LoveAndCoding) [inline templates/share/site_header.html]: "Variabless" got an extra "s" at some point

File: templates/genes/genes_list.html [hunk 2]
```diff
@@ -50,6 +51,11 @@ <h1 class="page-header pull-left">Saved Gene Favorites</h1>
                     {% endfor %}
                 </tbody>
             </table>
+            {% else %}
+            <p class="center">You don't have any gene favorite yet.</p>
```
Comment (LoveAndCoding) [inline templates/genes/genes_list.html]: May want to do

> You don't have any **Gene Favorites** yet.

File: templates/GenespotRE/dashboard.html [hunk 3]
```diff
@@ -37,12 +37,12 @@ <h1 class="page-header">Your Dashboard</h1>
...
                     <li class="item">
                         <!--TODO: get bar chart-->
-                        <img class="img-fluid" src="{% static 'img/scatter-plot.png' %}">
+                        <img class="img-fluid" src="{% static 'img/histogram.png' %}">
                         <p>Scatter Plot</p>
                     </li>
                     <li class="item">
                         <!--TODO: get bar chart-->
-                        <img class="img-fluid" src="{% static 'img/violin-plot.png' %}">
+                        <img class="img-fluid" src="{% static 'img/histogram.png' %}">
```
Comment (LoveAndCoding) [inline templates/GenespotRE/dashboard.html:45]: Did you mean to change these two?

Comment (litopia) [inline templates/GenespotRE/dashboard.html:45]: Yes. The violin-plot and scatter-plot images I gathered are not good. I think they should change them anyway.

File: app.yaml [hunk 4]
```diff
@@ -16,7 +16,7 @@
 #
 ###
 
-runtime: custom #python27 #custom
+runtime: custome #python27 #custom
```
Comment (LoveAndCoding) [inline app.yaml]: Extra "e" here now ;)

Comment (litopia) [inline app.yaml]: Lol, long day. :(

**Pull request merged**